Formula/container-diff.rb

```diff
@@ -0,0 +1,34 @@
+class ContainerDiff < Formula
+  desc "Diff your Docker containers"
+  homepage "https://github.com/GoogleCloudPlatform/container-diff"
+  url "https://github.com/GoogleCloudPlatform/container-diff/archive/v0.5.1.tar.gz"
+  sha256 "2de7635f485314ad8506222783c615620a4a7c32b0d39215df70f1b8964c871f"
+
+  head do
```

Comment:
`head "https://github.com/GoogleCloudPlatform/container-diff.git"`

Reply:
Going to add this to my _"I wonder if we can audit this"_ list locally. Single-line `head do` blocks have definitely made a comeback amongst contributors lately.

Reply:
it's more than single-lined
```
head do
  url "foo.git",
      :branch => "bar"
end
```

Reply:
and `:using`

Reply:
Yeah, essentially anything with nothing but a `url` inside a `head do` block is what I meant, more specifically.

Reply:
I'll leave all that between you and rubocop. :see_no_evil:

Reply:
I loathe working with that thing 🙈. Great tool, dubious usability/syntax choices.

Reply:
yup. I'd rather chase contributors around about this than deal with that thing